source/layouts/layout.erb

```diff
@@ -15,6 +15,21 @@
     <!--[if gte IE 9]><!--><%= stylesheet_link_tag :fonts, media: :all %><!--<![endif]-->
 
     <%= javascript_include_tag :application %>
+    <% if config.analytics != '' %>
```

Comment:
I wonder if it would be nicer having `.present?` instead of checking if the string is not empty?

Reply:
Ah I didn’t know that was a thing, will swap it out

Reply:
It's a method part of activesupport which middleman inherits so it should be fine. Its just a wrapper around `!blank?` but read nicer 👍